### ISSUE TITLE
Decorate status of deployed integrations to indicate when running

### ISSUE THREAD

**Issue by bfitzpat** [camel-tooling/vscode-camelk#55]
Some other extensions (such as the Kubernetes extension) offers a nice decorator to indicate when a pod is not quite ready yet (a little red indicator vs. a green one saying "ready").

It takes a bit when you start a new integration before you can actually view the log. It would be good to monitor the state of the new integration (and show the state change in the Camel-K output channel). 

We could also simply watch and wait for the state to change to "running" as complete or something else in error before populating the integrations view. But I think the decorator would be a better hint -- perhaps yellow as an "in progress" marker, red for "error" and green for "running"?

**Comment by bfitzpat** [comment]
Would also be nice to add more status bar indication... 
* Start an Integration
* Stop an integration
* Refresh the view (already done)
* Follow a log

**Comment by bfitzpat** [comment]
A few improvements in the works...
![image](https://user-images.githubusercontent.com/530878/62299924-2181d780-b433-11e9-8104-0c260f1bd7eb.png)

**Comment by apupier** [comment]
that's cool.
Keeping it as close as possible as VS Code Kubernetes extension is providing a good User Experience.

**Comment by apupier** [comment]
> Would also be nice to add more status bar indication...

for the status bar, I see it less important. I already have the bar almost full with the various extensions so not sure it will really help users.
![image](https://user-images.githubusercontent.com/1105127/62301169-5f551f00-b478-11e9-8cbf-a783e4e2d135.png)

**Comment by paoloantinori** [comment]
I like it, and to drag this slightly off topic:
one of the most recent enhancements in VSCode is the ability to turn off elements of the status bar:

https://code.visualstudio.com/updates/v1_36#_hide-individual-status-bar-items

**Comment by bfitzpat** [comment]
> I like it, and to drag this slightly off topic: one of the most recent enhancements in VSCode is the ability to turn off elements of the status bar: https://code.visualstudio.com/updates/v1_36#_hide-individual-status-bar-items

That's cool. I am not able to find an example of how to tap into that API anywhere, but opened a question https://stackoverflow.com/questions/57313451/how-do-we-tap-into-hiding-individual-status-bar-items-in-our-extension

**Comment by bfitzpat** [comment]
> for the status bar, I see it less important. I already have the bar almost full with the various extensions so not sure it will really help users.

It does get a little busy. These little indicators pop up and then disappear, so they're not quite as persistent -- but I do get your point. Hopefully we can find a way to shut them off per @paoloantinori 's suggestion. But I like having some kind of a visual indicator beyond the decorator that hints that something is in flight.

**Comment by bfitzpat** [comment]
To address @apupier 's concern about the crowded status bar for now, I've added a setting
![image](https://user-images.githubusercontent.com/530878/62326230-d5528980-b46a-11e9-81fb-f7ddc69798ef.png)

But I improved the integration status (fixed a parsing error) and now we have decorators. It's only green if it's "Running" - everything else is red. And there's currently no auto-refresh. User has to hit the Refresh button to update statuses.
![image](https://user-images.githubusercontent.com/530878/62326351-206c9c80-b46b-11e9-868d-72408be8eedc.png)

Not sure if we need an auto-refresh on this or not. WDYT?
